### ISSUE TITLE
Possibility to bind admin rights to a shortcut or a profile

### ISSUE THREAD

**Issue by Murazaki** [Eugeny/tabby#5867]
**Is your feature request related to a problem? Please describe.**
Right now admin rights are only accessible through contextual menu

**Describe the solution you'd like**
Solution could be multiple :
* open an admin console with a shortcut

**Describe alternatives you've considered**
* create an admin profile (might be already the case but not obvious in the interface)
* duplicate or turn the current console to admin
* give a modifier key for shortcuts for opening a console with admin rights

**Additional context**

**Comment by Eugeny** [comment]
You already can set a profile to run as admin, this what you mean? 
<img width="524" alt="image" src="https://user-images.githubusercontent.com/161476/156523191-13a3efda-73ab-41b5-8517-1c9b3c32006b.png">

**Comment by Murazaki** [comment]
> You already can set a profile to run as admin, this what you mean? <img alt="image" width="524" src="https://user-images.githubusercontent.com/161476/156523191-13a3efda-73ab-41b5-8517-1c9b3c32006b.png">

You're right, I totally missed that. For me it was more visible to change env variables and disable dynamic tab title than run as admin. Might be good to highlight those options with bigger buttons and a different color ?

Also I might be wondering it we could have an option for admin regardless of the profile, admin rights tend to clutter the profile list (and I ended up just using admin with cmder for instance, for convenience).

**Comment by mpacholke** [comment]
I am on 1.0.181 but the option is missing

<img width="783" alt="image" src="https://user-images.githubusercontent.com/97809255/178481693-a687b9a8-8f77-4105-a318-e3da53e88266.png">

**Comment by Eugeny** [comment]
@mpacholke admin tabs are only supported in W10 build 17692 and later.

**Comment by mpacholke** [comment]
Ok, thanks. This is Server 2016. Are there any restrictions concerning server editions?

**Comment by Eugeny** [comment]
No, but admin tabs rely on ConPTY which is not available before build 17692

**Comment by Chealer** [comment]
@Murazaki: As this is an ITS, each ticket should emphasize the problem, in particular in the title. Please retitle with a short description of the issue. I suggest:
> New profile screen: “Run as administrator“ could be more prominent

Tabby 1.0.230 still uses the funky widget pictured for this preference. A checkbox would surely be harder to miss.

**Comment by Murazaki** [comment]
You're right, I retitled it.
